Docs/development/jasp-building-guide.md

```diff
@@ -80,10 +80,13 @@ The directory structure should be as follows:
 		- libarchive.a
 		- libz.a
 
- 0. Clone the JASP repository into a folder of your choice. Our default choice is *~/desktop/JASP/*, this will create the folder */jasp-desktop* and QT will make the build-JASP-.... directory in this folder.
- 1. Create the folder */Frameworks* in the folder of your choice and unzip the **R 3.3.3** files into there
- 2. Unzip **boost 1.64.0** files in jasp-desktop
- 3. Unzip the **boost 1.64.0 binaries, libarchive binaries** files and put them into the */build-JASP-....* directory that QT creates. 
+ 0. **Clone** the JASP repository into a folder of your choice. Our default choice is *~/desktop/JASP/* and cloning results in the creating of *~/desktop/JASP/jasp-desktop*.
+ 1. **XCode**: Qt on OS X relies on XCode to function, you can install this through the App Store. It's easiest if you install this, run it, accept the license agreement, and then close it down before installing Qt.
+ 2. **Qt**: building JASP on OS X is pretty robust, but for the moment we're restricted to version 5.4.0. We will upgrade to a newer version soon.
```

Comment:
Can you add the following as a Note: ? (It seems to happen for Xcode version 8+)

For xcode 8 and above, for accepting the license, Qt (v5.4) tries to find `xcrun` instead of `xcodebuild`, and the following has to be done,
1. Open the file: `Qt_install_folder/5.4/clang_64/mkspecs/features/mac/default_pre.prf`
2. Replace the line `isEmpty($$list($$system("/usr/bin/xcrun -find xcrun 2>/dev/null")))` with `isEmpty($$list($$system("/usr/bin/xcrun -find xcodebuild 2>/dev/null")))`

Reply:
Thanks man, this is very helpful. See the change.